### PR TITLE
Po alert button

### PR DESCRIPTION
**Pull request by shruthipatil11**
Alert and button components

### CODE REVIEW

File: src/app/docs/docs.html [hunk 1]
```diff
@@ -247,11 +246,224 @@ <h3>Badge on Icons</h3>
...
+             </div>
+
+             <!-- Danger alert -->
+             <div class="alert alert--danger" role="alert">
+               <span><i class="fas fa-exclamation-triangle"></i></span>A danger
+               alert
+             </div>
+
+             <!-- Warning alert -->
+             <div class="alert alert--warning" role="alert">
```
Comment (rahulyadav139) [inline src/app/docs/docs.html:324]: Great work. You wrote the role attribute which helps screen readers to read this element as defined.

File: src/app/components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+.link:hover {
+  color: var(--GREEN-DARK);
+}
+
+.link.disabled {
+  background-color: var(--WHITE);
+  color: var(--DISABLED--GREY-DARK);
+}
+
+.btn__link {
```
Comment (rahulyadav139) [inline src/app/components/button/button.css:84]: class names are quite simple and easy to understand. Great work.

**Review by rahulyadav139 — APPROVED** [review]

File: src/app/docs/docs.html (hunk 1 above)
Comment (shruthipatil11) [reply src/app/docs/docs.html:324]: Thanks Rahul

File: src/app/components/button/button.css (hunk 2 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css:84]: Yep, following BEM

File: src/app/components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.btn--secondary:hover,
+.btn--outlined.btn--secondary:hover {
+  background-color: var(--GREEN-LIGHT);
+  color: var(--WHITE);
+}
+
+.disabled {
+  cursor: none;
```
Comment (muskanbagrecha) [inline src/app/components/button/button.css:34]: Change cursor type.

File: src/app/components/button/button.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+.btn--outlined.btn--secondary {
+  border: 1px solid var(--GREEN-LIGHTER);
+}
+
+.btn--outlined.disabled {
+  border: 1px solid var(--DISABLED--GREY-DARK);
+  color: var(--DISABLED-GREY);
+}
+
+.alert span,
```
Comment (muskanbagrecha) [inline src/app/components/button/button.css]: Is this meant to be here? You have this piece of CSS in alerts as well.

File: src/app/components/button/button.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+.btn__icon span {
+  margin-right: 1rem;
+}
+
+.btn__floating {
+  border: 3px solid whitesmoke;
```
Comment (muskanbagrecha) [inline src/app/components/button/button.css]: Use hex code instead of name colors. And use variables.

File: src/app/components/button/button.css [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  border-radius: 50%;
+  cursor: pointer;
+  box-shadow: 0.2rem 0.2rem 0.5rem 0.1rem var(--GREY);
```
Comment (muskanbagrecha) [inline src/app/components/button/button.css]: Such units can be given in px. They're quite small and box shadow is not necessarily something that will be made responsive.

**Review by muskanbagrecha — CHANGES_REQUESTED** [review]

**Review by Ananyamadhu08 — APPROVED** [review]
looks good, can go ahead and merge 👍

**Comment by muskanbagrecha** [comment]
Great work. Loved the naming convention.

File: src/app/components/button/button.css (hunk 4 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css]: yeah, this is common css between alerts and button.I didn't know where exactly to place them.I didn't even find it appropriate to place in utilities as well .Any suggestions

File: src/app/components/button/button.css (hunk 3 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css:34]: added pointer events: none to disabled button and in html added disabled attribute.

In disabled link : aria-disabled="true" is added and href attribute is removed as per disabled button conventions

File: src/app/components/button/button.css (hunk 4 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css]: separated css

File: src/app/components/button/button.css (hunk 5 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css]: sure, thanks for info. I have replaced them

File: src/app/components/button/button.css (hunk 6 above)
Comment (shruthipatil11) [reply src/app/components/button/button.css]: changed to px

**Review by muskanbagrecha — APPROVED** [review]
Reviewed Changes. Good to merge.

**Comment by shruthipatil11** [comment]
Merging alert and button components to development

**Pull request merged**